packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -149,6 +160,33 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
           }
         </div>
         <FocusZone>
+          {
+            showWeekNumbers ?
+              <table
+                className={ css('ms-DatePicker-weekNumbers', styles.weekNumbers, 'ms-DatePicker-table', styles.table) }
+              >
+                <tbody>
```

Comment:
>tbody [](start = 17, length = 5)

Do we need accessibility for the week numbers? This can be done as a separate task if you run out of time, we just need to make sure to open the WI in VSO, thanks!